gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -3698,7 +3869,7 @@ subroutine GFS_physics_driver                                 &
 
           else      ! ras version 2
 
-! DH* this code not yet in CCPP (belongs to GFS_DCNV_generic_pre?)
+!## CCPP ##* This code not yet in CCPP Note: Likely belongs in rascnv_pre.
```

Comment:
I have moved this calculation inside RAS itself.  There is no need for rascnv_pre